ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -38,7 +48,38 @@ def start(self) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("Cluster start operation is not implemented.")
+        _, _, _rc = self._execute.run_cmd(const.PCS_CLUSTER_STATUS, check_error=False)
+        if _rc != 0:
```

Comment:
where is the code for starting cluster on this node?

Reply:
As discussed over code, I did the code changes